components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -126,7 +135,7 @@ export default {
   },
   args: {
     rootClass: "spectrum-PickerButton",
-    label: "Select a Country",
+    placeholder: "Select a Country",
```

Comment:
```suggestion
    placeholder: "Select a country",
```

Minor nit - the styleguide says we are using sentence case rather than title case so I try to align the demos with that whenever I can.